pages/admin/permit-holders.tsx

```diff
@@ -1,16 +1,344 @@
+        }
+        .Selectable .DayPicker-Day {
+          border-radius: 0 !important;
+        }
+        .Selectable .DayPicker-Day--start {
+          border-top-left-radius: 50% !important;
+          border-bottom-left-radius: 50% !important;
+        }
+        .Selectable .DayPicker-Day--end {
+          border-top-right-radius: 50% !important;
+          border-bottom-right-radius: 50% !important;
+        }
+      `}</style>
+    </Helmet>
+  );
+}
 
 // Internal home page
 export default function PermitHolders() {
+  const [permitStatusFilter, setPermitStatusFilter] = useState('All');
+  const [userStatusFilter, setUserStatusFilter] = useState('All');
+  const [range, setRange] = useState<RangeModifier>({ from: undefined, to: undefined });
+
+  const permitStatusOptions = [
+    'All',
+    'In Progress',
+    'Completed',
+    'Pending',
+    'Expiring Soon',
+    'Expired',
+    'Rejected',
+  ];
+  const userStatusOptions = ['All', 'Active', 'Inactive'];
+
+  const handleDayClick: DayPickerProps['onDayClick'] = day => {
+    setRange(DateUtils.addDayToRange(day, range));
+  };
+
+  const modifier = { start: range.from || undefined, end: range.to || undefined };
+
+  const dateRangeString = () => {
+    if (!range.to) {
+      return 'YYYY-MM-DD - YYYY-MM-DD';
+    }
+    if (range.from && range.to) {
+      return `${range.from.toLocaleDateString('en-CA')} - ${range.to.toLocaleDateString('en-CA')}`;
+    }
+  };
+
   return (
     <Layout>
       <GridItem colSpan={12}>
-        <Text textStyle="display-xlarge">Permit Holders</Text>
+        <Flex align="left" marginBottom="32px">
+          <Text textStyle="display-xlarge">Permit Holders</Text>
+        </Flex>
+        <Box border="1px solid" borderColor="border.secondary" borderRadius="12px">
+          <Box padding="20px 24px">
+            <Flex marginBottom="20px">
+              <Menu>
+                <MenuButton
+                  as={Button}
+                  variant="outline"
+                  rightIcon={<ChevronDownIcon />}
+                  marginRight="12px"
+                  color="text.secondary"
+                  borderColor="border.secondary"
+                >
+                  <MenuText name={`Permit Status`} value={permitStatusFilter} />
+                </MenuButton>
+                <MenuList>
+                  <DropDownItems
+                    items={permitStatusOptions}
+                    setStateCallback={setPermitStatusFilter}
+                  />
+                </MenuList>
+              </Menu>
+              <Menu>
+                <MenuButton
+                  as={Button}
+                  variant="outline"
+                  rightIcon={<ChevronDownIcon />}
+                  marginRight="12px"
+                  color="text.secondary"
+                  borderColor="border.secondary"
+                >
+                  <MenuText name={`User Status`} value={userStatusFilter} />
+                </MenuButton>
```

Comment:
Need to confirm this with designers but could we make the menu buttons a fixed size? Currently, the width of the menu box changes according to the options in the dropdown menu selected.

For example, this is what we see in Figma (there is more space after the "all")
![image](https://user-images.githubusercontent.com/60051956/127282833-b6c0aa33-b45c-4b4b-bcb2-4c356c8ac351.png)

Compared to what we have right now (no space after "all" and button size is fit to text length):
![image](https://user-images.githubusercontent.com/60051956/127282744-8af1851e-05dd-40f7-a4ad-fa0e2eda1292.png)

Reply:
Good catch! Made it fixed width